docs/source/changelog.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 =========
 
+**v0.2.4**
+    * Entry Point Test
```

Comment:
Can we change this to "Added Featuretools Entry Point"

Reply:
updated